Intersect (Core)/GameObjects/EquipmentProperties.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.Json.Serialization;
+using Intersect.GameObjects.ItemRange;
+
+namespace Intersect.GameObjects;
+public class EquipmentProperties
+{
+    public EquipmentProperties()
+    {
+    }
+
+    public EquipmentProperties(Guid descriptorId)
+    {
+        ItemDescriptorId = descriptorId;
+    }
+
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public Guid Id { get; private set; }
+
+    public Guid ItemDescriptorId { get; set; }
+
+    [ForeignKey(nameof(ItemDescriptorId))]
+    public ItemBase Item { get; set; }
+
+    public List<StatRange> StatRanges { get; set; }
```

Comment:
This may need to be `virtual` and I still want us to look into using a `Dictionary<Stat, StatRange?>` property bag if it's possible.

Reply:
Need to do more reading, not entirely sure what a property bag is still. If I'm understanding correctly, though, assuming EF handles things correctly we could then ditch `StatRange.cs` and _only_ have `ItemRange.cs`? If so, future expansion could look like...

```cs
Dictionary<Stat, ItemRange> StatRanges
Dictionary<Effect, ItemRange> EffectRanges
Dictionary<Vital, ItemRange> VitalRanges
```

?

Reply:
I could see this being possible. I would expect it to create columns like `StatRanges_Strength`, `VitalRanges_Health`. Would need to verify by generating a migration and checking the output.